GGCNN_Grasp[] Grasp_list
